mopidy/m3u/translator.py

```diff
@@ -74,11 +74,14 @@ def parse_m3u(file_path, media_dir=None):
     - Lines starting with # are ignored, except for extended M3U directives.
     - Track.name and Track.length are set from extended M3U directives.
     - m3u files are latin-1.
+    - m3u8 files are utf-8
     """
     # TODO: uris as bytes
+    file_encoding = 'utf-8' if file_path.endswith('.m3u8') else 'latin1'
```

Comment:
Our URIs are always Unicode (with non-ASCII charcs URI encoded), and our file paths are always bytes. To check that the file path ends with `.m3u8`, you must use `.endswith(b'.m3u8')`.